src/hooks/voltz-config/config.ts

```diff
@@ -0,0 +1,271 @@
+import { providers } from 'ethers';
+
+import { NetworkConfiguration } from './types';
+
+export const networkConfigurations: { [key: string]: NetworkConfiguration } = {
+  mainnet: {
+    factoryAddress: '0x6a7a5c3824508D03F0d2d24E0482Bea39E08CcAF',
+    apply: true,
+    pools: [
+      // aUSDC pools
+      {
+        name: 'aUSDC_v1',
+        id: '0xae16bb8fe13001b61ddb44e2ceae472d6af08755', // vamm address
```

Comment:
do we have to update the UI every time we create a pool?

Reply:
yes, we do for mainnet. But we'd have needed anyway because of whitelists.